# Waypoint message for global path
int32 id
float64 x_m
float64 y_m
float64 s_m
float64 d_m
float64 psi_rad
float64 kappa_radpm
float64 vx_mps
float64 d_left
float64 d_right
